man/format_ci.Rd

```diff
@@ -1,5 +1,5 @@
 % Generated by roxygen2: do not edit by hand
-% Please edit documentation in R/format_CI.R
+% Please edit documentation in R/format_ci.R
```

Comment:
I [fixed this](https://github.com/easystats/parameters/commit/ca1ee6148c32cc14566196f4ae7088144c67d30d) yesterday, but seemed to be overwritten again? `format_ci` is wrong here, `format_CI` is correct.

Reply:
in my computers I have format_ci, this issue comes I think from windows where if you replace a name file by chancing the case it is not properly recognized as a change. one workaround is to rename the file to smt else first, and then re-rename it to the desired name

Reply:
That was too confusing for me. :-D
The file is names `format_CI.R`, right?
https://github.com/easystats/parameters/blob/master/R/format_CI.R

Reply:
Mmmmh on github yes... for some reason on my computer it's format_ci lol, in that case I'll change on my computer? Or should we change on github?

Reply:
Let's change it on github for consistency